Logger.js

```diff
@@ -11,7 +11,7 @@ function initialize(debugMode) {
 }
 
 /**
- * Error - this is a issues that needs attention right now.
+ * Error - this is an issues that needs attention right now.
```

Comment:
Does the logs obey to the log level conventions. Errors are for things that need immediate attention because SDK won’t work. Warnings are for potential errors in the future. Info is to indicate publicly available method calls. Debug should give information about the inner working of those calls (and checks of those calls). Verbose should print noisy internal workings.